### PR TITLE
Update about-shared-mailboxes.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the change in sharedmailbox size after july 2018 as there could be org's still using the old provisioned shared mailbox.

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9436

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/email/about-shared-mailboxes.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ Before you [create a shared mailbox](create-a-shared-mailbox.md), here are some
...
 
+
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/about-shared-mailboxes.md]:
```suggestion
```

File: microsoft-365/admin/email/about-shared-mailboxes.md [hunk 2]
```diff
@@ -63,6 +63,10 @@ Before you [create a shared mailbox](create-a-shared-mailbox.md), here are some
...
+> [!NOTE]
+> Prior to July 2018 all unlicensed sharedmailboxes were provisioned with 100Gb mailbox size. for more information, see [Correcting Shared Mailbox provisioning and sizing](https://techcommunity.microsoft.com/t5/exchange-team-blog/correcting-shared-mailbox-provisioning-and-sizing/ba-p/607991)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/about-shared-mailboxes.md]:
```suggestion
> Prior to July 2018 all unlicensed shared mailboxes were provisioned with a size of 100 GB. For more information, see [Correcting Shared Mailbox provisioning and sizing](https://techcommunity.microsoft.com/t5/exchange-team-blog/correcting-shared-mailbox-provisioning-and-sizing/ba-p/607991).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@SKjerland Please merge this PR. Thanks

**Pull request merged**